public/docs/_examples/pipes/dart/lib/exponential_strength_pipe.dart

```diff
@@ -0,0 +1,28 @@
+library pipes_examples.exponential_strength_pipe;
+
+import 'dart:math' as math;
+
+import 'package:angular2/angular2.dart';
+
+/*
+* Raise the value exponentially
+* Takes a value that defaults to 0 and an exponent argument that defaults to 1.
+* Checks for value to be a string or number.
+* Usage:
+*   value | exponentialStrength:exponent
+* Example:
+*   {{ 2 |  exponentialStrength:10}}
+*   formats to: 1024
+*/
+
+@Pipe(name: 'exponentialStrength')
+@Injectable()
+class ExponentialStrengthPipe {
```

Comment:
Shouldn't a pipe implement `PipeTransform`?
Why would a pipe need the `@Injectable()` annotation while directives and components don't?